orion/core.go

```diff
@@ -499,3 +502,39 @@ func GetDefaultServerWithConfig(config Config) Server {
 		config: config,
 	}
 }
+
+func GetDefaultServerWithHandlerOptions(name string, opts ...DefaultServerOption) Server {
```

Comment:
do you think we can just extend "GetDefaultServer"? `opts ...DefaultServerOption` is an optional arguments so it won't cause any breaking change.